packages/core/src/domain/configuration/tags.ts

```diff
@@ -7,8 +7,8 @@ export const TAG_SIZE_LIMIT = 200
 declare const __BUILD_ENV__SDK_VERSION__: string
 
 export function buildTags(configuration: Configuration): string[] {
-  const { env, service, version, datacenter } = configuration
-  const tags = [buildTag('sdk_version', __BUILD_ENV__SDK_VERSION__)]
+  const { env, service, version, datacenter, sdkVersion, variant } = configuration
+  const tags = [buildTag('sdk_version', sdkVersion ?? __BUILD_ENV__SDK_VERSION__)]
```

Comment:
@amortemousque we couldn't always send `browser_sdk_version` as it is currently used in Web UI to detect if the event is coming from a WebView.